DataFormats/PatCandidates/interface/VIDCutFlowResult.h

```diff
@@ -0,0 +1,60 @@
+#ifndef __DataFormats_PatCandidates_VIDResult_H__
+#define __DataFormats_PatCandidates_VIDResult_H__
+
+#include <unordered_map>
```

Comment:
this is breaking ROOT5 build.
Please fix.
if actual compilation in root5 is desired, I can suggest  to start from slava77:CMSSW_7_5_ROOT5_X_2015-06-22-2300/sign566/pull9659-bp and then apply fixes there (to be cherry-picked back to the 76X version); a working build is CMSSW_7_5_ROOT5_X_2015-06-22-2300

... if this feature is not required in 75X or earlier; we can live without root5 support in 76X.